reporting-engine-worker/src/main/java/org/ihtsdo/termserver/scripting/reports/release/PackageComparisonReport.java

```diff
@@ -1,35 +1,56 @@
 package org.ihtsdo.termserver.scripting.reports.release;
 
-import java.io.IOException;
+import java.io.*;
 import java.util.*;
 
 import org.ihtsdo.otf.exception.TermServerScriptException;
 import org.ihtsdo.termserver.scripting.ReportClass;
 import org.ihtsdo.termserver.scripting.reports.TermServerReport;
 import org.snomed.otf.scheduler.domain.*;
 import org.snomed.otf.scheduler.domain.Job.ProductionStatus;
-import org.snomed.otf.script.dao.ReportSheetManager;
 
 /**
  * 
  */
 public class PackageComparisonReport extends TermServerReport implements ReportClass {
-	
-	public static final String A_RELEASE = "A Release";
-	public static final String B_RELEASE = "B Release";
-	
-	//public static final String PREV_DEPENDENCY = "Previous Dependency";
-	//public static final String THIS_DEPENDENCY = "This Dependency";
+
+	public static final String PREVIOUS_RELEASE = "Previous Release";
+	public static final String PREVIOUS_RELEASE_ZIP = "Previous Release Package";
+	public static final String CURRENT_RELEASE = "Current Release";
+	public static final String CURRENT_RELEASE_ZIP = "Current Release Package";
+	public static final String PUBLISHED_FOLDER = "Published Folder";
+
+	private final String scriptName = "run_compare.sh";
+	private String[] scriptArguments;
 	
 	public static void main(String[] args) throws TermServerScriptException, IOException {
 		Map<String, String> params = new HashMap<>();
+		params.put(PREVIOUS_RELEASE, "EE Release 2021-11-30");
```

Comment:
This is a surprise... does the string here relate to the SRS folder name?   This doesn't look like the metadata we have against CodeSystems.    I think you need to add a description to those parameters.  See https://github.com/IHTSDO/reporting-engine/blob/develop/reporting-engine-worker/src/main/java/org/ihtsdo/termserver/scripting/reports/TextDefinitionsPresentAndMissing.java#L40 for an example of this.    Actually I don't know that these are displayed in the UI yet - I'll check with @peterduff

Reply:
@pgwilliams this is what scripts are asking for - some names for 2 packages being compared. As far as I understand, that is some free text the user gives, they are used to name the folder where all the comparison takes place. I will add a comment. Ultimately we can do without them I think. But I kept them in order not to change the scripts unnecessarily.